hardware_interface/include/hardware_interface/resource_manager.hpp

```diff
@@ -493,6 +507,8 @@ class ResourceManager
    */
   void set_on_component_state_switch_callback(std::function<void()> callback);
 
+  const std::string & get_urdf() const;
```

Comment:
```suggestion
  const std::string & get_robot_description() const;
```
I think we technically already support SDF but even if that is not that case we should not tie ourselves to URDF

Reply:
Agree with this comment!

Reply:
A very valid point. I will do that right away